spec/active_record_american_gladiator_spec.rb

```diff
@@ -45,7 +45,7 @@
       order_amounts = []
 
       # Changeable Start
-      users = User.first(3)
+      users = User.includes(:orders).first(3)
```

Comment:
I prefer to use `.limit` here because `.first` has if/else logic to see whether you've passed a numeric parameter or not and THEN call limit anyway. Might as well skip all that and call `.limit` yourself :)